src/server/ocpi/ocpi-services-impl/ocpi-2.1.1/OCPISessionsService.ts

```diff
@@ -257,6 +257,7 @@ export default class OCPISessionsService {
         endedAt: new Date(session.last_updated),
         consumption: transaction.currentConsumptionWh,
         instantPower: Math.round(transaction.currentConsumption),
+        instantAmps: transaction.currentConsumption / 230, // Utils convert method was updated. Needs to be handled by Serge.
```

Comment:
It's not that hard to just merge with the master-qa branch and follow the API change of the method: it's just one argument added to the conversion function.

Reply:
This is consumption from OCPI, we have no voltage available here on charging station or site area.
@ellavas Why in consumption there is neither siteID nor siteAreaID props ?